src/Features/CSharp/Portable/CodeFixes/GenerateMethod/GenerateMethodCodeFixProvider.cs

```diff
@@ -80,6 +83,12 @@ protected override SyntaxNode GetTargetNode(SyntaxNode node)
                 return memberBindingExpression.Name;
             }
 
+            var assignmentExpression = node as AssignmentExpressionSyntax;
```

Comment:
It's because the CS0123 issue has a squiggly under the entire line and the AbstractGenerateMemberCodeFixProvider looks up the tree from the start of the squiggly.